src/main/asciidoc/override/eventbus.adoc

```diff
@@ -24,7 +24,8 @@ This lookup happens after the system codecs are examined, i.e a default codec
 for `io.vertx.core.buffer.Buffer` will not be used.
```

Comment:
I think this is maybe a bad example, because `Buffer` is a interface, not a class. Changing the example and maybe even adding a own comment about that interfaces are not considered when trying to find a codec would be advisable. What do you think?

Reply:
Maybe:

> This lookup happens after the system codecs (these include the java primitive types, `Buffer`, `JsonObject` and `JsonArray`) are examined, i.e a default codec for `io.vertx.core.json.JsonObject` will not be used.